README.md

```diff
@@ -1841,6 +1842,8 @@ Update Time, five active automations, webhooks.
   * [Tiledesk](https://tiledesk.com) - Create chatbots and conversational apps. Bring them omnichannel: from your website (live chat widget) to WhatsApp. Free plan with unlimited chatbots.
   * [Versionfeeds](https://versionfeeds.com) — Custom RSS feeds for releases of your favorite software. Have the latest versions of your programming languages, libraries, or loved tools in one feed. (The first 3 feeds are free)
   * [videoinu](https://videoinu.com) — Create and edit screen recordings and other videos online.
+  * [TinySnap](https://tinysnap.app) - A tiny screenshot extension that effortlessly transforms screenshots, text, and code into stunning images.
+  * [Jingle Bio](https://jingle.bio) - Create an impressive developer portfolio by featuring your projects, repositories, and skills all in one concise bio page.
```

Comment:
This says "get started for free" but says nothing about what that means or whats not free, also that its in early beta, happy to add it later once there are details.

Reply:
@ripienaar , its look like earlybird.im & jingle.bio both are same service , Because jingle.bio is redirecting to earlybird.im.

Their **free plan is very limited** . According to their Plan page - **Free plan has only 100 Monthly visits** 
And their landing page also says that they are in **Private Beta**

Feel Free to Correct Me If I'm Wrong , @dearroy 

![Screenshot from 2023-12-21 10-52-26](https://github.com/ripienaar/free-for-dev/assets/97863234/f3513d34-8310-4cb1-8aba-ef7f7bfc0831)

![Screenshot from 2023-12-21 11-03-20](https://github.com/ripienaar/free-for-dev/assets/97863234/7adbe0ef-faa3-404a-a329-14d1913d2089)

Reply:
Been long enough without response here so will just close it